src/components/typography/Label.tsx

```diff
@@ -0,0 +1,36 @@
+import { Text, StyleSheet } from "react-native";
+
+import { global } from "@/constants/DesignTokens";
+
+type LabelProps = { size?: "large" | "medium" | "small" | "extraSmall" };
+
+type TextProps = LabelProps & Text["props"];
+
+export default function Label(props: TextProps) {
+  const { style, size = "large", ...otherProps } = props;
+
+  return <Text style={[styles.label, styles[size], style]} {...otherProps} />;
+}
```

Comment:
You can do that inside the props, no need for another line

```suggestion
export default function Label({ style, size = "large", ...otherProps }: TextProps) {
  return <Text style={[styles.label, styles[size], style]} {...otherProps} />;
}
```